code-challenges.js

```diff
@@ -29,23 +76,56 @@ var nums1 = [3, 56, 90, -8, 0, 23, 6]
 var nums2 = [109, 5, 9, -59, 8, 24]
 // Expected output: [-59, 109]
 
+//create a function that takes in an array
+const minMax = (array) => {
+    // Return the minimum and maximum numbers from the array. Use Array.from to convert the 2 numbers into an array that we can return.
+    return Array.from([(Math.min(...array)), (Math.max(...array))])
+}
```

Comment:
LOOK AT YOU! All in ONE return AND using .from AND using that spread operator AND Math.max and Math.min. 

IM SO PROUD OF YOU!